tasks.py

```diff
@@ -51,7 +51,7 @@ def is_truthy(arg):
 namespace.configure(
     {
         "nautobot_ssot": {
-            "nautobot_ver": "2.3.1",
+            "nautobot_ver": "2.3.16",
```

Comment:
I don't think we want to be doing this.

Reply:
Yeah...I'd agree. Not too sure why I initially changed this. Reverted!